src/scm/Model.ts

```diff
@@ -296,7 +296,7 @@ export class Model implements Disposable {
             const uri = Uri.file(clientFile);
             const resource: Resource = new Resource(uri, change, action);
 
-            if (change === 'default change') {
+            if (change.startsWith('default')) {
```

Comment:
`p4 opened` will output `default change`
but `p4 fstat` will output `... change default`

Don't ask me why 😞 

So, yes, your change is valid.